ethereum/datastructures/src/main/java/tech/pegasys/teku/datastructures/util/SimpleOffsetSerializer.java

```diff
@@ -229,6 +236,12 @@ public static Bytes serializeVariableCompositeList(
     }
   }
 
+  public static ViewRead deserialize(Bytes bytes, ViewType sszViewType) {
```

Comment:
Looks like this can be private:
```suggestion
  private static ViewRead deserialize(Bytes bytes, ViewType sszViewType) {
```